### PR TITLE
Add uncut-sans font

### PR DESCRIPTION
**Pull request by lumoe**
PR for Issue: https://github.com/fontsource/fontsource/issues/622

### Font Name

Uncut Sans

### Link to font source

https://github.com/kaspernordkvist/uncut_sans/

### Link to font license

https://github.com/kaspernordkvist/uncut_sans/blob/main/LICENSE.txt

### CODE REVIEW

**Comment by lumoe** [comment]
I am not entirely sure why the Vercel deployment fails, I also cannot lookup the details in the log.  

Any insights if my PR contains any mistakes?

**Comment by jwr1** [comment]
You can ignore vercel, it's only used for previewing website changes.

File: fonts/other/uncut-sans/files/rename.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import os
```
Comment (ayuhito) [inline fonts/other/uncut-sans/files/rename.py]: Looks good to me! I just think we need to remove this one file and we should be good to go!

**Review by ayuhito — CHANGES_REQUESTED** [review]

**Comment by lumoe** [comment]
Removed the file. Thank you for having a look at it!

**Review by ayuhito — APPROVED** [review]

**Comment by ayuhito** [comment]
Thanks for the PR! You should see the changes published within an hour.

**Pull request merged**